csharp/ql/integration-tests/all-platforms/autobuild_slnx/autobuild_slnx.slnx

```diff
@@ -0,0 +1,3 @@
+<Solution>
+  <Project Path="proj1/proj1.csproj" />
```

Comment:
I know there is a comment in `test.py` already, but perhaps add one here as well saying that the exclusion of `proj2` is deliberate.